modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc

```diff
@@ -28,29 +28,60 @@ endif::openshift-dedicated,openshift-rosa[]
 
 .Procedure
 
-. From the *Developer* perspective in the {product-title} web console, select *Observe* -> *Metrics*.
+. In the *Developer* perspective of the {product-title} web console, click *Observe* and go to the *Metrics* tab.
 
 . Select the project that you want to view metrics for in the *Project:* list.
 
-. Select a query from the *Select query* list, or create a custom PromQL query based on the selected query by selecting *Show PromQL*. The metrics from the queries are visualized on the plot.
+. To add one or more queries, perform any of the following actions:
++
+|===
+|Option |Description
+
+|Select an existing query.
+|From the *Select query* drop-down list, select an existing query.
+
+|Create a custom query.
+|Add your Prometheus Query Language (PromQL) query to the *Expression* field.
+
+As you type a PromQL expression, autocomplete suggestions appear in a drop-down list. These suggestions include functions, metrics, labels, and time tokens.
+Use the keyboard arrows to select one of these suggested items and then press Enter to add the item to your expression. Move your mouse pointer over a suggested item to view a brief description of that item.
+
+|Add multiple queries. |Click *Add query*.
+
+|Duplicate an existing query. |Click the options menu {kebab} next to the query, then choose *Duplicate query*.
+
+|Disable a query from being run. |Click the options menu {kebab} next to the query and choose *Disable query*.
+|===
+
+. To run queries that you created, click *Run queries*. The metrics from the queries are visualized on the plot. If a query is invalid, the UI shows an error message.
 +
 [NOTE]
 ====
-In the Developer perspective, you can only run one query at a time.
+* When drawing time series graphs, queries that operate on large amounts of data might time out or overload the browser. To avoid this, click *Hide graph* and calibrate your query by using only the metrics table. Then, after finding a feasible query, enable the plot to draw the graphs.
+
+* By default, the query table shows an expanded view that lists every metric and its current value. Click the *˅* down arrowhead to minimize the expanded view for a query.
 ====
 
-. Explore the visualized metrics by doing any of the following:
+. Optional: The page URL now contains the queries you ran. To use this set of queries again in the future, save this URL.
```

Comment:
Not something I'd stop merge review over, but per [the ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps) the first sentence of each step needs to be written in the imperative mood, even if it has "Optional:" at the beginning. Idk what an appropriate fix would be like, but maybe the first sentence can be treated as redundant and removed:
```suggestion
. Optional: Save the page URL to use this set of queries again in the future.
```